### ISSUE TITLE
[Feature request] Plantuml support just like mermaid

### ISSUE THREAD

**Issue by hanye9895** [Tencent/cherry-markdown#86]
For most cases, mermaid should be enough, but for some people, they would like to use plantuml

** What does the proposed API look like**
Just like mermaid:
```
import Cherry from 'cherry-markdown/dist/cherry-markdown.core';

const registerPlugin = async () => {
  const [{ default: CherryPlantumlPlugin }, plantuml] = await Promise.all([
    import('cherry-markdown/src/addons/cherry-code-block-plantuml-plugin'),
    import('plantuml'),
  ]);
  Cherry.usePlugin(CherryPlantumlPlugin, {
    plantuml, // pass in plantuml object
  });
};

registerPlugin().then(() => {
  //  Plug-in registration must be done before Cherry is instantiated
  const cherryInstance = new Cherry({
    id: 'markdown-container',
    value: '# welcome to cherry editor!',
  });
});
```

**Comment by hanye9895** [comment]
seems already done

**Comment by hanye9895** [comment]
But with a further look, seems it do not contain in dist folder~

**Comment by sunsonliu** [comment]
@jiawei686 Do you have any ideas?

**Comment by hanye9895** [comment]
> @jiawei686 Do you have any ideas?
It seems plantuml addons were not included in dist folder
<img width="1141" alt="addon issue" src="https://user-images.githubusercontent.com/68373171/150719552-3868d554-4c44-447c-b8b8-2a9e74751593.png">

And here is my usage of plantuml (just like mermaid)
```
const [
          { default: CherryMermaidPlugin },
          { default: mermaid },
          { default: CherryPlantumlPlugin },
          { default: plantuml }
        ] = await Promise.all([
          import('cherry-markdown/dist/addons/cherry-code-block-mermaid-plugin'),
          import('mermaid'),
          import('cherry-markdown/dist/addons/cherry-code-block-plantuml-plugin'),
          import('plantuml')
        ]);
        if (!window.hasPlugin) {
          Cherry.usePlugin(CherryMermaidPlugin, {
            mermaid 
          });
          Cherry.usePlugin(CherryPlantumlPlugin, {
            plantuml
          });
          window.hasPlugin = true;
        }
```

So, I try to include that in the dist folder~ 
Or, is there another usage of plantuml without import these addons?
